src/app/components/iframe/Iframe.jsx

```diff
@@ -37,7 +37,7 @@ export default class Iframe extends Component {
     }
 
     render() {
-        return <iframe id="iframe" className="preview__iframe" src={this.props.path}></iframe>;
+        return <iframe id="iframe" key={this.props.previewLanguage} className="preview__iframe" src={this.props.path}></iframe>;
```

Comment:
I don't like having to use key, feels hacky but not sure how else I can get it to behave the way I want.